rstudio/c9s-python-3.9/Dockerfile

```diff
@@ -50,6 +50,7 @@ COPY rsession.conf /etc/rstudio/rsession.conf
 # package installation 
 RUN dnf install -y libsodium-devel.x86_64 libgit2-devel.x86_64 libcurl-devel harfbuzz-devel.x86_64 fribidi-devel.x86_64 cmake "flexiblas-*"
 RUN R -e "install.packages('Rcpp')"
+RUN if [-n = $NEEDPACKAGE]; then R -e "install.packages(c('tidyverse', 'tidymodels', 'plumber', 'vetiver', 'devtools'))" && R -e "install_versions('tidyverse', version = '2.0.0', 'tidymodels', version = '1.1.1', 'vetiver', version = '0.2.5', 'plumber', version = '1.2.1', 'devtools', verison = '2.4.5' )"; else exit 0
```

Comment:
Also, curious question - it's interesting how this installation of packages work. First an install (without specifying of particular version) and then we attach to a particular versions :thinking:

I have no idea here, but based on [this SO comment](https://stackoverflow.com/a/17082609), it should be possible to do something like this:

```
install.versions(c('checkpoint', 'devtools'), c('0.3.3', '1.6.1'))
```

Again, I didn't try it in any way, so don't take me extremely serious here.